### ISSUE TITLE
SubscriptionChange does not have typescript interface

### ISSUE THREAD

**Issue by ciocan** [wildbit/postmark.js#81]
``` 
await serverClient.createWebhook({
      Url: `https://webhook.com`,
      MessageStream: 'newsletter',
      Triggers: {
        Delivery: {
          Enabled: true,
        },
        Bounce: {
          Enabled: true,
          IncludeContent: false,
        },
        SpamComplaint: {
          Enabled: true,
          IncludeContent: false,
        },
        Open: {
          Enabled: true,
          PostFirstOpenOnly: true,
        },
        Click: {
          Enabled: true,
        },
        // @ts-ignore
        SubscriptionChange: {
          Enabled: true,
        },
      },
    })
```

The **SubscriptionChange** does not have a typescript interface and prompts linter error.
However, the webhook is created with subscription change enabled.

**Comment by ibalosh** [comment]
Thanks @ciocan 

sorry for late response, will see to check this out soon

**Comment by ibalosh** [comment]
We have fixed this issue in version 2.7.6
